tlog.spec

```diff
@@ -34,14 +30,13 @@ License:        GPLv2+
 %endif
 
 URL:            https://github.com/Scribery/%{name}
-Source:         %{url}/releases/download/v%{version}/%{name}-%{version}.tar.gz
+Source0:        %{url}/releases/download/v%{version}/%{name}-%{version}.tar.gz
+Source1:        %{name}-tmpfiles.conf
 
-BuildRequires:  autoconf
-BuildRequires:  automake
-BuildRequires:  libtool
```

Comment:
These are required when building from the source tree, as the autotools scripts are not committed in version control.